libs/pipelines/track_online_inertial.cpp

```
@@ -463,28 +466,40 @@ bool SolverSfMInertial::predict_pose(Isometry3T& pose) const {
 
 void SolverSfMInertial::set_verbose(bool verbose) { verbose_ = verbose; }
 
+// Reports whether the IMU stream covers the frame interval reliably. See the matching helper in
+// imu_fusion_context.cpp for the rationale — short frame intervals previously read as 33%-drop
+// false positives from normal IMU phase jitter.
 bool check_imu_drops(sba_imu::IMUPreintegration& preint, const imu::ImuCalibration& calib, int64_t start_time_ns,
```

Clang (14.0.6)

[warning] use a trailing return type for this function

(modernize-use-trailing-return-type)